substrate/client/rpc-spec-v2/src/chain_head/subscription/inner.rs

```diff
@@ -272,15 +221,15 @@ impl RegisteredOperation {
 	/// Returns the number of reserved elements for this permit.
 	///
 	/// This can be smaller than the number of items requested via [`LimitOperations::reserve()`].
+	#[allow(unused)]
```

Comment:
remove?!

Reply:
It looks like we are not discarding items from the storage query anymore?

Reply:
This was only used to truncate storage query iterations and it's not needed anymore when we have backpressure i.e, adapt to client to produce new items.

However, we are still using the semaphore to limit the number of operations

So this ` num_reserved` is removed now, ok?

Reply:
This is where it was used: https://github.com/paritytech/polkadot-sdk/pull/5741/files#diff-85e700332cc343b498a3d97dd807aa72af6e0abfcad6046645d351d96010943bL436-L440

Reply:
Yep makes sense, thanks Niklas 🙏